application-development/helloworld-springboot-auth/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+security:
+  oauth2:
+    client:
+      clientId: 667713450471-6u065o1ph801fr80kqa0popiudsvathb.apps.googleusercontent.com
+      clientSecret: cqR_pUlx1BjEBumoE0o69WkF
+      accessTokenUri: https://www.googleapis.com/oauth2/v4/token
+      userAuthorizationUri: https://accounts.google.com/o/oauth2/v2/auth
+      authenticationScheme: form
+      scope:
+        - openid
+        - email
+        - profile
+        - https://www.googleapis.com/auth/cloud-platform
```

Comment:
should remove this "cloud-platform" scope as it's very wide and shouldn't e needed, and why openid?  I would think just email and profile is necessary.
thanks